labs/1.md

```diff
@@ -0,0 +1,52 @@
+<a id="markdown-introduction" name="introduction"></a>
+# Introduction
+
+This demo is a simple deployment of [Federation Operator](https://operatorhub.io/operator/alpha/federation.v0.0.10) on two OpenShift
```

Comment:
I would potentially remove the MongoDB information here. It seems somewhat confusing because we talk about 2 clusters but then cite the video with 3 clusters. With the task we are trying to do in this introduction the mongo part doesn't seem to matter.